environments/aks/ptlsbox.tfvars

```diff
@@ -1,6 +1,6 @@
 clusters = {
   "00" = {
-    kubernetes_cluster_version = "1.26"
+    kubernetes_cluster_version = "1.26.3"
```

Comment:
Is this needed? I thought 1.26 - will determine latest 1.26.X

Reply:
Doesn't seem to. [Here](https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=393388&view=logs&j=9529ead6-39ae-5319-1740-9d2035106bd6&t=89b2e481-5d95-5be1-d64b-e101e3d88784&l=358) shows it trying to perform a downgrade

Reply:
@adusumillipraveen FYI^?

Reply:
Curiously, sbox does not exhibit the same behaviour despite also being on 1.26.3. None of the other environments are complaining about 1.25. Perhaps it's a random bug that can be ignored

Reply:
Reverted the patch version back to `1.26` and it's still proposing the same updates on ptlsbox.